docs/03-tutorials/00-serverless/svls-13-override-runtime-image

```diff
@@ -0,0 +1,95 @@
+---
+title: Override runtime image
+---
+
+This tutorial show how to build custom runtime image and override functions base image with thee built one
+
+## Prerequisites
+
+Before you start, make sure you have these tools installed:
+
+- Kyma installed on a cluster
+
+## Steps
+
+Follows these steps:
+
+1. Follow this example to build python's custom runtime image.
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="cli">
+  Kyma CLI
+  </summary>
+
+2. Export these variables:
+
+    ```bash
+    export NAME={FUNCTION_NAME}
+    export NAMESPACE={FUNCTION_NAMESPACE}
+    export RUNTIME_IMAGE={RUNTIME_IMAGE_WITH_TAG}
+    ```
+
+3. Create your local development workspace using built image:
+
+    ```bash
+    mkdir {FOLDER_NAME}
+    cd {FOLDER_NAME}
+    kyma init function --name $NAME --namespace $NAMESPACE --runtime-image-override $RUNTIME_IMAGE
```

Comment:
Please use --runtime in the example as it is important to match the override
You want to kyma CLI to generate handler and requirements files depending on the matching override